src/Core/Auth/UserFeatures/UserKey/Implementations/RotateUserKeyCommand.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Bit.Core.Auth.Models.Data;
+using Bit.Core.Repositories;
+using Bit.Core.Services;
+using Microsoft.AspNetCore.Identity;
+
+namespace Bit.Core.Auth.UserFeatures.UserKey.Implementations;
+
+public class RotateUserKeyCommand : IRotateUserKeyCommand
```

Comment:
question: should this be internal?
```suggestion
internal class RotateUserKeyCommand : IRotateUserKeyCommand
```